coalib/misc/Exceptions.py

```diff
@@ -20,10 +15,10 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        LogPrinterMixin().log_exception(log_message, exception)
```

Comment:
Right now, implemented using `LogHelper.log_exception` function instead of class. Will update it in future if required to be merged.
By the way, Is there any advantage of implementing as a single method instead of static method of a class ?